src/components/DevelopmentServerPane/DevelopmentServerPane.js

```diff
@@ -163,6 +189,20 @@ const TerminalWrapper = styled.div`
   }
 `;
 
+const ButtonWrapper = styled.label`
+  display: flex;
+`;
+
+const ButtonStrip = styled.div`
+  padding: 5px;
+  margin-left: auto;
```

Comment:
Curious what this `margin-left` is doing? Is this a way of pushing the buttons to the right?

I think the way I would do this is have `justify-content: space-between` on `ButtonWrapper`.

Not a huge deal though, I can always tweak after merging :)